memgpt/local_llm/json_parser.py

```diff
@@ -5,7 +5,19 @@
 from memgpt.errors import LLMJSONParsingError
 
 
-def extract_first_json(string):
+def replace_escaped_underscores(string: str):
```

Comment:
Noticed that Groq API w/ mixtral was outputting a lot of escaped underscores, so added some extra JSON parsing code to strip